packages/react-themeless-provider/package.json

```diff
@@ -8,10 +8,6 @@
     "@griffel/react": "^1.5.14",
     "@swc/helpers": "~0.5.2"
   },
-  "devDependencies": {
```

Comment:
we dont specify devDeps ( everything is consumed from monorepo root ) - single version policy